charts/consul/templates/_helpers.tpl

```diff
@@ -73,6 +73,13 @@ as well as the global.name setting.
           {{ "{{" }}- end -{{ "}}" }}
 {{- end -}}
 
+{{- define "consul.vaultBootstrapTokenConfigTemplate" -}}
+|
+          {{ "{{" }}- with secret "{{ .Values.global.acls.bootstrapToken.secretName }}" -{{ "}}" }}
+          acl { tokens { initial_management = "{{ "{{" }}- {{ printf ".Data.data.%s" .Values.global.acls.bootstrapToken.secretKey }} -{{ "}}" }}" }}
+          {{ "{{" }}- end -{{ "}}" }}
+{{- end -}}
```

Comment:
The consul servers are configured to have the `initial_management` token that is stored in vault. This is an alternative to calling the `/v1/acl/bootstrap` api to get the bootstrap token. I think in our case this is safe because vault will store this config in tmpfs.